### PR TITLE
Master branch should work now

### PR DESCRIPTION
**Pull request by mzer0-yu**
It should have been merged yesterday, but was found not.

Notify me if something wrong with dev branch.

### CODE REVIEW

File: core/includes/cs_page.h [hunk 1]
```diff
@@ -1,7 +1,40 @@
-#include "pageProcesser.h"
+#ifndef CS_PAGE_H
+#define CS_PAGE_H
+#endif
```
Comment (AntonioCS) [inline core/includes/cs_page.h]: Why are the header guards not used correctly?

File: core/includes/cs_page.h [hunk 2]
```diff
@@ -1,7 +1,40 @@
...
+
+/*
+  page carrier
+*/
+
+#define FileTypeErr 0
```
Comment (AntonioCS) [inline core/includes/cs_page.h]: defines normally are all CAPS

File: core/includes/page_queue.h [hunk 3]
```diff
@@ -1,4 +1,8 @@
-#include "pageProcesser.h"
+#ifndef PAGE_QUEUE_H
+#define PAGE_QUEUE_H
```
Comment (AntonioCS) [inline core/includes/page_queue.h]: Same issue with the header guards

File: core/includes/cs_page.h (hunk 2 above)
Comment (mzer0-yu) [inline core/includes/cs_page.h]: Do you mean # DEFINE ???

Comment (AntonioCS) [inline core/includes/cs_page.h]: No, I meant FILE_TYPE_ERR

**Comment by mitghi** [comment]
@mzer0-yu @AntonioCS 
This [issue](https://github.com/luohaha/CSpider/issues/34) is still present.

**Comment by AntonioCS** [comment]
@mitghi The issues I pointed out weren't fixed and as you stated issue [#34](https://github.com/luohaha/CSpider/issues/34) is still present.

Maybe wait and do a better review of the code before merging it.

File: core/includes/cs_page.h (hunk 2 above)
Comment (mzer0-yu) [inline core/includes/cs_page.h]: That's...a personal style

**Comment by mitghi** [comment]
@AntonioCS I reverted the pull request, now it must be fine. 
@mzer0-yu Can you please check if your build works ? because some typedefs are missing and this commit exactly creates the issue [#34](https://github.com/luohaha/CSpider/issues/34).

**Comment by mzer0-yu** [comment]
@mitghi I am sorry I don't have a build environment now, something wrong with my laptop.

I am checking.

**Comment by mzer0-yu** [comment]
OK. Finally I know it. @luohaha Changed the name of cs_page to cs_page_struct, so it fails.

File: core/includes/cs_page.h (hunk 2 above)
Comment (AntonioCS) [inline core/includes/cs_page.h]: That will just confuse people. Please stick with the standard way to do constants in C

**Comment by mitghi** [comment]
@mzer0-yu Great.

**Comment by mzer0-yu** [comment]
@mitghi I don't know which branch I am supposed to edit. Please edit the cs_page.h
> change the name of cs_page_struct to cs_page
> change the name of page_queue_struct to page_queue
under #3

**Comment by mzer0-yu** [comment]
The reason why fails is because luohaha rewrites the name of cs_page and page_queue but I don't know, neither had a build environment now.

**Comment by mitghi** [comment]
@mzer0-yu I am checking it, check CS.h . These two typedefs are there.
    typedef struct cs_page_struct cs_page;
    typedef struct cs_page_queue_struct cs_page_queue;

**Comment by luohaha** [comment]
@mzer0-yu You are right, I forget to tell you that I rewrites the name of cs_page and page_queue.

**Comment by mzer0-yu** [comment]
@luohaha Never mind.
@mitghi But cs_page.c doesn't have included CS.h

**Comment by mzer0-yu** [comment]
So the way is, either including the CS.h, or changing the name to original cs_page instead of cs_page_struct.

**Comment by mitghi** [comment]
@mzer0-yu Alright, I will do it then commit it to dev branch. When build pass we can merge it to master. Thanks.

**Pull request merged**